ChangeLog.zephyr.md

```diff
@@ -3,6 +3,9 @@
 Need to take care to not break these changes when updating pico-sdk.
 
 ## Patch List:
+  - [#10] pico-sdk: Disabling sanity check the IRQ status
```

Comment:
```suggestion
  - [#10] pico-sdk: Disabling DMA routines and sanity check the IRQ status
```

Crypto subsystem does support DMA though, so if this feature becomes desired, then should it be implemented in the Zephyr driver part? Couldn't the existing HAL code be used for DMA operations?

Reply:
Reasonable, most of that code just calls into HAL DMA methods